doc/whats_new.rst

```diff
@@ -323,6 +323,14 @@ Enhancements
      (`#5540 <https://github.com/scikit-learn/scikit-learn/pull/5540>_`)
      By `Andreas Müller`_.
 
+   - When unpickling a scikit-learn estimator in a different version than the one
+     the estimator was trained with, a ``UserWarning`` is raised, see `the documentation
+     on model persitance
+     <http://scikit-learn.org/stable/modules/model_persistence.html#security-maintainability-limitations>`_
```

Comment:
No, I meant an internal anchor link, so that a link in `/dev/` will point to `/dev/` rather than stable.